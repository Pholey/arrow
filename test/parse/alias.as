
// Alias
type c_int = int32;
type c_llong = int128;
