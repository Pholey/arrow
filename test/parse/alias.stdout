{
  "tag": "Module",
  "span": "test/parse/alias.as:1-4",
  "statements": [
    {
      "tag": "TypeAlias",
      "span": "test/parse/alias.as:3:1-19",
      "name": "c_int",
      "target": {
        "tag": "TypeName",
        "span": "test/parse/alias.as:3:14-19",
        "name": {
          "tag": "Name",
          "span": "test/parse/alias.as:3:14-19",
          "text": "int32",
          "type_arguments": []
        }
      }
    },
    {
      "tag": "TypeAlias",
      "span": "test/parse/alias.as:4:1-22",
      "name": "c_llong",
      "target": {
        "tag": "TypeName",
        "span": "test/parse/alias.as:4:16-22",
        "name": {
          "tag": "Name",
          "span": "test/parse/alias.as:4:16-22",
          "text": "int128",
          "type_arguments": []
        }
      }
    }
  ]
}
